lib/webapp.js

```diff
@@ -199,6 +199,10 @@ function getPresetHeader(name, preset, req) {
       return `default-src 'self' ${req.hostname};`;
     } else if (preset == 'frame-strict') {
       return `frame-src 'self' ${req.hostname};`;
+    } else if (preset == 'script-nonce') {
+      let nonce=crypto.randomBytes(16).toString('hex');
```

Comment:
I am wondering whether the approach with a random string would be suitable for HA mode when there are multiple app-servers running.

Reply:
Good point, probably not.